### PR TITLE
docs: update internal docs

### PR DESCRIPTION
**Pull request by Conaclos**
## Summary

Rename Rome to Biome in internal docs.
Rename LSP `rome.rome` to `biomejs.biome`.

## Test Plan

CI

### CODE REVIEW

File: crates/rome_cli/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # `rome_cli`
 
-Rome's main binary distribution, exposes the command line interface defined in
+Main binary distribution, exposes the command line interface defined in
```
Comment (ematipico) [inline crates/rome_cli/README.md:3]: The main

File: crates/rome_css_parser/README.md [hunk 2]
```diff
@@ -1,36 +1,23 @@
 <p align="center">
-	<picture>
-		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/rome/brand/main/PNG/logo_white_yellow_transparent.png" width="700">
-		<img alt="Rome's logo depicting an ancient Roman arch with the word Rome to its side" src="https://raw.githubusercontent.com/rome/brand/main/PNG/logo_transparent.png" width="700">
-	</picture>
+	<img alt="Biome - Toolchain of the web" width="400" src="https://raw.githubusercontent.com/biomejs/resources/main/biome-logo-slogan.svg"/>
```
Comment (ematipico) [inline crates/rome_css_parser/README.md:2]: Don't we have a light/dark version?

**Review by ematipico — APPROVED** [review]

**Comment by ematipico** [comment]
make sure to run `just codegen-website` to update the code generated files

File: crates/rome_css_parser/README.md (hunk 2 above)
Comment (Conaclos) [reply crates/rome_css_parser/README.md:2]: I think the current logo is a good fit for both theme?

**Pull request merged**